plugins/pulp_docker/plugins/registry.py

```diff
@@ -377,7 +377,10 @@ def get_manifest(self, reference):
         # set the headers for first request
         request_headers['Accept'] = schema2
         response_headers, manifest = self._get_path(path, headers=request_headers)
-        digest = self._digest_check(response_headers, manifest)
+        if response_headers.get(content_type_header) != schema2:
```

Comment:
@pcreech the reason i did not `response_headers.get(content_type_header) == schema1` because schema1 can be signed and not signed and i would need to check for 2 mediatypes

Reply:
Since we are specifically weeding out the schema2 case, I'm happy with this conditional